### PR TITLE
Feat/tcpchannel

### PR DESCRIPTION
**Pull request by samartse**
@BenMoon

### CODE REVIEW

**Comment by samartse** [comment]
@BenMoon

File: pymepix/pymepix_connection.py [hunk 1]
```diff
@@ -86,14 +88,23 @@ def data_thread(self):
...
+
 
             
 
     def __init__(self,
                  spidr_address=(cfg.default_cfg["timepix"]["tpx_ip"], 50000),
                  src_ip_port=('192.168.1.1', 8192), 
-                 pipeline_class=PixelPipeline):
+                 pipeline_class=PixelPipeline,
+                 chan_address=('127.0.0.1', 5056)):
```
Comment (BenMoon) [inline pymepix/pymepix_connection.py]: We should probably remove this hard-coded address, as well as we should remove it for `src_ip_port` and rather issue a warning or take if from the `default.yaml`.

**Review by BenMoon — CHANGES_REQUESTED** [review]
Please review my comment, and maybe we should also take out all the other ecomo*.yaml files from the repo.

File: pymepix/pymepix_connection.py (hunk 1 above)
Comment (samartse) [reply pymepix/pymepix_connection.py]: Done.

**Pull request merged**